ansible/templates/grafana-labs/config.alloy.j2

```diff
@@ -0,0 +1,151 @@
+    action        = "keep"
+  }
+}
+
+loki.source.journal "logs_integrations_integrations_node_exporter_journal_scrape" {
+  max_age       = "24h0m0s"
+  relabel_rules = discovery.relabel.logs_integrations_integrations_node_exporter_journal_scrape.rules
+  forward_to    = [loki.write.grafana_cloud_loki.receiver]
+}
+
+local.file_match "logs_integrations_integrations_node_exporter_direct_scrape" {
+  path_targets = [{
+    __address__ = "localhost",
+    // __path__    = "/var/log/{syslog,messages,*.log}",
```

Comment:
I am excluding syslogs, etc. for now. This will avoid blowing up our quota – at least thats what I hope.